core/tx_list.go

```diff
@@ -241,8 +250,11 @@ func newTxList(strict bool) *txList {
 func (l *txList) Add(tx *types.Transaction) (bool, *types.Transaction) {
 	// If there's an older better transaction, abort
 	old := l.txs.Get(tx.Nonce())
-	if old != nil && old.GasPrice().Cmp(tx.GasPrice()) >= 0 {
-		return false, nil
+	if old != nil {
+		threshold := new(big.Int).Div(new(big.Int).Mul(old.GasPrice(), big.NewInt(100+minPriceBumpPercent)), big.NewInt(100))
```

Comment:
old `gasPrice==0` => `threshold` = 0.
old `gasPrice==3` => `threshold`= 3

(with minbumppercent at 25). 

Should  add some handling of the lower extremes.